lib/node_modules/@stdlib/math/base/special/fresnel/test/test.js

```diff
@@ -0,0 +1,236 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var PINF = require( '@stdlib/math/constants/float64-pinf' );
+var NINF = require( '@stdlib/math/constants/float64-ninf' );
+var EPS = require( '@stdlib/math/constants/float64-eps' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var randu = require( '@stdlib/math/base/random/randu' );
+var fresnel = require( './../lib' );
+
+
+// FIXTURES //
+
+var smallPositive = require( './fixtures/c/cephes/small.json' );
+var mediumPositive = require( './fixtures/c/cephes/medium.json' );
+var largePositive = require( './fixtures/c/cephes/large.json' );
+var hugePositive = require( './fixtures/c/cephes/huge.json' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.equal( typeof fresnel, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function computes in-place the Fresnel integral (small positive values)', function test( t ) {
+	var delta;
+	var tol;
+	var C;
+	var S;
+	var x;
+	var y;
+	var z;
+	var i;
+
+	z = [ 0.0, 0.0 ];
+	x = smallPositive.x;
+	S = smallPositive.S;
+	C = smallPositive.C;
+
+	for ( i = 0; i < x.length; i++ ) {
+		y = fresnel( z, x[i] );
+		t.equal( y, z, 'returns a reference to out' );
+		if ( y[0] === S[ i ] ) {
```

Comment:
Be consistent here with array bracket spacing. e.g., `y[ 0 ]` or `S[i]`.

Reply:
The same applies elsewhere in this file.

Reply:
Don't care which is chosen; use best judgment; just that there is consistency.

Reply:
Opted for `S[i]`.

Reply:
✅